plugins/backend/rag-ai-backend/src/service/RagAiController.ts

```diff
@@ -64,6 +71,9 @@ export class RagAiController {
   }
 
   createEmbeddings = async (req: Request, res: Response) => {
+    const { token } = await this.tokenManager.getToken();
```

Comment:
These requests are never sent anywhere further so I don't think it's strictly necessary to add the additional headers in this case. Same for the endpoints below